controllers/users.controller.js

```diff
@@ -0,0 +1,10 @@
+
+
```

Comment:
Where's the import of getUsers? Also the model and controller functions are the wrong way round. Have a look at my naming conventions I posted in the cohort group earlier in the week.

Reply:
I don't know how is this mixed up  the merge keeps changes